pages/index.tsx

```diff
@@ -103,7 +104,8 @@ export async function getStaticProps() {
     props: {
       officers: JSON.parse(JSON.stringify(officers)),
       chapters: JSON.parse(JSON.stringify(chapters)),
-    }
+    },
+    revalidate: globals.revalidate.journal,
```

Comment:
should this be `globals.revalidate.home`? Also we can probably increase the time on this page since officers won't be updated frequently, maybe `43200` for at most twice a day